fixtures/look/unique-2-1.json

```diff
@@ -0,0 +1,50 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Unique 2.1",
+  "categories": ["Smoke"],
```

Comment:
"From the finest mist to a thick haze, similar to fog, any desired effect can be achieved."  The manual uses the word "fog" 30 times, and the word "haze" 29 times.  Is this a hazer or a smoke/fog machine?  Eh...